Makefile

```
@@ -59,6 +59,14 @@ uv-pyavd-utils-install: pyavd-utils-build ## Build and install pyavd-utils Pytho
 uv-pyavd-utils-editable-install: ## Build and install pyavd-utils as editable
```

let's add this to the editable installs as well

```suggestion
uv-pyavd-utils-editable-install: check-cargo ## Build and install pyavd-utils as editable
```